src/main/java/io/codelex/Test/Exercise2/Test.java

```diff
@@ -0,0 +1,24 @@
+package io.codelex.Test.Exercise2;
+
+public class Test {
+    public static void main(String[] args) {
+        Apple apple = new Apple(1, "Apple1");
```

Comment:
Šis vispār kompilējas? Jo kods pilnībā neatbilst klasēm, kuras esi iekomitojis.

Reply:
Šo klasi esmu izdzēsis. Tā nav jāņem vērā.